filebeat/processor/add_kubernetes_metadata/matchers.go

```diff
@@ -77,7 +77,7 @@ const containerIdLen = 64
 const podUIDPos = 5
 
 func (f *LogPathMatcher) MetadataIndex(event common.MapStr) string {
-	if value, ok := event["source"]; ok {
+	if value, err := event.GetValue("log.source"); err == nil {
```

Comment:
@exekias Could you confirm that this is the expected field?

Reply:
that looks correct, I understand we are release this with 7.0, isn't it? No plans to backport it to 6.x

Reply:
This is in the list of required tasks for Suricata, which ships in 6.5. Maybe I misunderstood and this change was only required in the long term.

Reply:
When we backport this to 6.5 we will need to "undo" the `source -> log.source` change and do something with the suricata `source.ip` like `ecs.source.ip`.

Reply:
I think it would make sense to split this in 2 PRs, 1 to backport (without breaking changes) and the one that brings the breaking changes (not backported)

Reply:
My assumption was also this is only for master. +1 on splitting it up then.

Reply:
Thanks. I've updated the PR removing the breaking changes. Once this is merged and backported I will submit the breaking filebeat changes to master/7.0.